src/components/screens/Tour.js

```diff
@@ -98,7 +96,7 @@ export const Tour = ({ data, navigation, route }) => {
           </View>
         )}
 
-        {!!augmentedReality && <AugmentedReality navigation={navigation} tourID={id} />}
+        {!!tourStops?.length && <AugmentedReality {...{ id, navigation }} />}
```

Comment:
now we will have tour stops data here through the tour query and make the same query for the data again in `AugmentedReality`? i suggest not doing it this way. if there is already data for tour stops we can pass and use them like before.
only for settingsscreen we can make a new query. therefore we should skip the useQuery hook if not `onSettingsScreen`. you will find such `skip` already in the code for other places i guess.

Reply:
Yeah, I'll fix that.

Reply:
https://github.com/ikuseiGmbH/smart-village-app-app/pull/608/commits/afa7f49f3c7659b48f0bc217d6e549966e443c24